### DISCUSSION TITLE
Why the customisation in Funnel chart is limited?

### DISCUSSION THREAD

**Discussion by codebreaker-v2-0** [recharts/recharts#3833] [Q&A]
I'm just curious to know why there isn't much support for the Funnel Chart. There is neither a way to change the orientation of the chart nor any support for using custom shapes. I just wanted to know why there isn't much support for the Funnel Charts.

**Comment by ckifer** [comment]
Answered in the issue but probably just because funnel was added later than the other components and shape was never thought to be added as a feature. 

We can use your issue as a feature request.

**Comment by ckifer** [comment]
going to close this discussion since we have https://github.com/recharts/recharts/issues/3832
